src/components/Modals/Send/views/QrCodeScanner.tsx

```diff
@@ -1,24 +1,47 @@
-import { Button, ModalBody, ModalCloseButton, ModalFooter, ModalHeader } from '@chakra-ui/react'
-import { lazy, Suspense } from 'react'
-import { useFormContext } from 'react-hook-form'
-import { useTranslate } from 'react-polyglot'
-import { useHistory } from 'react-router-dom'
-import { SlideTransition } from 'components/SlideTransition'
-import { Text } from 'components/Text'
+import {
+  Alert,
+  AlertIcon,
+  Button,
+  Flex,
+  ModalBody,
+  ModalCloseButton,
+  ModalFooter,
+  ModalHeader
+} from '@chakra-ui/react'
+import {lazy, Suspense, useState} from 'react'
+import {useFormContext} from 'react-hook-form'
+import {useTranslate} from 'react-polyglot'
+import {useHistory} from 'react-router-dom'
+import {SlideTransition} from 'components/SlideTransition'
+import {Text} from 'components/Text'
 
-import { SendFormFields, SendInput } from '../Form'
-import { SendRoutes } from '../Send'
+import {SendFormFields, SendInput} from '../Form'
+import {SendRoutes} from '../Send'
+
+const PermissionError = 'Permission denied'
+
+enum ErrorMessages {
+  permissions = 'modals.send.errors.qrPermissions',
+  generic = 'modals.send.errors.generic'
+}
```

Comment:
I don't see the value of an enum here. I'd rather you put just the string right in the `setErrorMessage` calls (lines 41/43)

Reply:
Sorry, the PR was using draft mode because the work was in progress, the implementation did change a lot!

Reply:
I'm waiting for a response from the product team and I'll probably change the PR as ready for review :)